tests/unittests/test_currently_syncing.py

```diff
@@ -0,0 +1,109 @@
+import unittest
+from unittest import mock
+from tap_github.sync import (update_currently_syncing_repo, update_currently_syncing,
+                             get_ordered_stream_list, get_ordered_repos)
+
+class TestGetOrderedStreamList(unittest.TestCase):
+    """
+    Test `get_ordered_stream_list` function to get ordered list od streams
+    """
+
+    def test_for_interrupted_sync(self):
+        """Test if sync was interrupted"""
```

Comment:
Update/Add detailed comment as both test cases look similar and, also assertion is about list comparison & not around if sync is interrupted or not.